unblob/models.py

```diff
@@ -86,10 +86,13 @@ class Handler(Protocol):
 
     NAME: str
     YARA_RULE: str
+    # We need this, because not every match reflects the actual start
+    # (e.g. tar magic is in the middle of the header)
+    YARA_MATCH_OFFSET: int
```

Comment:
This `YARA_MATCH_OFFSET` is not added everywhere, please add those!
(while you were working on this branch, new `filesystem` handlers were introduced and they would fail because of this )